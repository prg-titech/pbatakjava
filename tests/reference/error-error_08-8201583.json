{
    "basename": "error-error_08-8201583",
    "cmd": "{infile}",
    "infile": "tests/error_08/build.sh",
    "infile_hash": "a4ea91bbbfa34ebea1c0cad55d1a4bdf5dc4de5f13f625d1b00523ed",
    "stdout": null,
    "stdout_hash": null,
    "stderr": "error-error_08-8201583.stderr",
    "stderr_hash": "355944dfbd1e667a173be4c15d724b22cd392ecdc4bc9f5dc2fa2b5e",
    "returncode": 4
}