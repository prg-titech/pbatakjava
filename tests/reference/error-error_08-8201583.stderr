tests/error_08/Test.java:4: error: no applicable method for copy()
